sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/OperationWithIdLiveTests.cs

```diff
@@ -54,10 +97,13 @@ public async Task BuildDocumentModelOperationCanParseOperationId(WaitUntil waitU
             var options = new BuildDocumentModelOptions(modelId, DocumentBuildMode.Template, source);
 
             Operation<DocumentModelDetails> operation = null;
+            string operationId = null;
 
             try
             {
                 operation = await client.BuildDocumentModelAsync(waitUntil, options);
+                operationId = operation.Id;
+
                 await operation.WaitForCompletionAsync();
```

Comment:
Could be a bit silly but we're making sure here that we're storing the operation ID before calling `WaitForCompletion`. Just want to make sure the ID is correctly populated in all stages of the long-running operation.